packages/Webkul/Admin/src/Resources/lang/ar/app.php

```diff
@@ -3630,6 +3630,10 @@
                     'shop-email-from'       => 'عنوان البريد الإلكتروني للمتجر',
                     'shop-email-from-tip'   => 'عنوان البريد الإلكتروني لهذه القناة لإرسال رسائل البريد الإلكتروني لعملائك',
                     'title'                 => 'إعدادات البريد الإلكتروني',
+                    'contact-email'         => 'عنوان البريد الإلكتروني للاتصال',
+                    'contact-email-tip'     => 'سيتم عرض هذا العنوان في أسفل رسائل البريد الإلكتروني الخاصة بك',
+                    'contact-name'          => 'اسم الاتصال',
+                    'contact-name-tip'      => 'سيتم عرض هذا الاسم في أسفل رسائل البريد الإلكتروني الخاصة بك',
```

Comment:
Please short the new added key as per the `email-settings` all keys.

Make sure the keys in the email settings are in `ascending order`.